modules/cloud-init/variables.tf

```diff
@@ -0,0 +1,74 @@
+variable "airgap_package_url" {
+  default     = ""
+  description = "The URL of an airgap package which contains a TFE release."
+  type        = string
+}
+
+variable "airgap_installer_url" {
+  default     = "https://install.terraform.io/installer/replicated-v5.tar.gz"
+  description = "The URL of an airgap package which contains the cluster installer."
+  type        = string
+}
+
+variable "application_config" {
+  description = "The application configuration."
+  type        = map(map(string))
+}
+
+variable "custom_ca_cert_url" {
+  default     = ""
+  description = "The URL of a certificate authority bundle which contains custom certificates to be trusted by the application."
+  type        = string
+}
+
+variable "distribution" {
+  default     = "ubuntu"
+  description = "The type of Linux distribution which will be running on the machines."
+  type        = string
+}
+
+variable "license_file" {
+  description = "The pathname of a Replicated license file for the application."
+  type        = string
+}
+
+variable "proxy_address" {
+  description = "The IP address of the Kubernetes API."
+  type        = string
+}
+
+variable "proxy_url" {
+  default     = ""
+  description = "The URL of a proxy through which application traffic will be routed."
+  type        = string
+}
+
+variable "ptfe_url" {
+  default     = "https://install.terraform.io/installer/ptfe-0.1.zip"
+  description = "The URL of the cluster installer tool."
+  type        = string
+}
+
+variable "release_sequence" {
+  default     = "latest"
+  description = "The sequence identifier of the TFE version to which the cluster will be pinned."
+  type        = string
+}
+
+variable "repl_cidr" {
+  default     = ""
+  description = "A custom IP address range over which Replicated will communicate, expressed in CIDR notation."
+  type        = string
+}
+
+variable "ssh_import_id_usernames" {
+  default     = []
+  description = "The usernames associated with SSH keys which will be imported from a keyserver to all machines."
```

Comment:
It might be nice as we're updating this to add some more info on what this means, specifically that it's really only supported on debian/ubuntu distributions and should be in the form of `lp:username` for a launchpad hosted public key, or `gh:username` for a github hosted public key.

Reply:
Based on the [cloud-init documentation](https://cloudinit.readthedocs.io/en/latest/topics/modules.html#ssh-import-id), it seems like the keyserver reference is optional but it's also not clear how this works without the reference. Should we just link to the underlying documentation rather than summarizing it and possibly missing some details?

Reply:
yeah I don't think that we want to add keyservers to this, but I agree that at least a link to the docs for usage would be good.

Reply:
Whoops, I missed this. Fixed by https://github.com/hashicorp/terraform-google-terraform-enterprise/pull/38/commits/89053669bdb79c31a3ed5950d5837d38ac2136ba.